PowerUp/app/src/main/res/layout-land/completed_game.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:gravity="right"
```

Comment:
You should not be using right gravity + margin as this will behave differently on each device. Instead, define the gravity as center and remove the margins. With this approach, you will probably not need to define this extra landscape layout file but just modify the original layout file so it works accross devices